charms/istio-pilot/src/charm.py

```diff
@@ -450,6 +469,15 @@ def _get_gateway_service(self):
         )
         return svc
 
+    def _handle_istio_pilot_relation(self):
+        """Handles the istio-pilot relation, sending information about the Istio daemon."""
+        istio_pilot_interface = self._get_istio_pilot_interface()
+        if istio_pilot_interface:
+            # If something is related, send the data.  Otherwise skip
+            istio_pilot_interface.send_data(
+                {"service-name": f"istiod.{self.model.name}.svc", "service-port": "15012"}
```

Comment:
always the same port?

Reply:
Yeah.  Currently we don't configure it in the istio install, so it is the default